ocis-reva/pkg/command/drivers.go

```diff
@@ -71,9 +71,11 @@ func drivers(cfg *config.Config) map[string]interface{} {
 			"share_folder": cfg.Reva.Storages.Local.ShareFolder,
 		},
 		"localhome": map[string]interface{}{
-			"root":         cfg.Reva.Storages.Local.Root,
-			"share_folder": cfg.Reva.Storages.Local.ShareFolder,
-			"user_layout":  cfg.Reva.Storages.Local.UserLayout,
+			"root":                cfg.Reva.Storages.Local.Root,
+			"share_folder":        cfg.Reva.Storages.Local.ShareFolder,
+			"user_layout":         cfg.Reva.Storages.Local.UserLayout,
+			"treetime_accounting": true,
+			"treesize_accounting": true,
```

Comment:
I think those settings have to go into the `ocis` section to make it work with the OCIS storage driver